wrappers/golang/curves/bls12377/ntt_test.go

```diff
@@ -201,6 +201,11 @@ func TestNttBatch(t *testing.T) {
 	}
 }
 
+func TestReleaseDomain(t *testing.T) {
```

Comment:
We need to ensure that this tests is a teardown part of the tests otherwise other tests that run after it will fail having no domain allocated